hist/hist/src/TH3.cxx

```diff
@@ -2358,6 +2358,11 @@ TH2D *TH3::DoProject2D(const char* name, const char * title, const TAxis* projX,
 ///  To exclude underflow and/or overflow (for both axis in case of a projection to a 1D histogram) use option "NUF" and/or "NOF"
 ///  With SetRange() you can have all bins except underflow/overflow only if you set the axis bit range as
 ///  following after having called SetRange:  axis->SetRange(1, axis->GetNbins());
+///
+///  NOTE 5: This function calls DoProject2D, which creates a new histogram (if same name does not already exists)
```

Comment:
Can you please change in note 2 (above): 
If an histogram of the same type and with the same name already exists,.......

Then in Note 5 just write (no need to mention DoProject2D) and the issue with same name that is discussed in Note2

NOTE 5: If TH1::AddDirectory was set to false, a new histogram is always created and the ownership of the returned pointer is delegated to the user. Be sure in this case to call 'delete` on it after it's no longer needed, to avoid memory leaks.

Reply:
Thanks. Done with slight modifications.